yml/dp02_dc2.yaml

```diff
@@ -7090,692 +7090,692 @@ tables:
     "@id": "#DiaObject.decl"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Declination coordinate of the position of the diaObject at time radecTai
   - name: diaObjectId
     "@id": "#DiaObject.diaObjectId"
     datatype: long
     mysql:datatype: BIGINT
-    description:
+    description: Unique id.
   - name: gPSFluxChi2
     "@id": "#DiaObject.gPSFluxChi2"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Chi^2 statistic for the scatter of gPSFlux around gPSFluxMean
   - name: gPSFluxErrMean
     "@id": "#DiaObject.gPSFluxErrMean"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Mean of the diaSource PSF flux errors
   - name: gPSFluxLinearIntercept
     "@id": "#DiaObject.gPSFluxLinearIntercept"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: y-intercept of a linear model fit to diaSource PSF flux vs time
   - name: gPSFluxLinearSlope
     "@id": "#DiaObject.gPSFluxLinearSlope"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Slope of a linear model fit to diaSource PSF flux vs time
   - name: gPSFluxMAD
     "@id": "#DiaObject.gPSFluxMAD"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Median absolute deviation of diaSource PSF flux. Does not include scale factor for comparison to sigma
```

Comment:
The second sentence is a bit context free here: should we explicitly state that scale factor in the description (1.4-whatever assuming normal errors)?

Should we say "sigma" in that last sentence, or "standard deviation" (the former is the field name to compare to, the latter is the description of that field).

Reply:
Maybe " Multiply by 1.4826 to compare to a normal standard deviation"?